apps/aehttp/priv/oas3.yaml

```diff
@@ -2351,15 +2354,15 @@ components:
         from_id:
           $ref: "#/components/schemas/EncodedPubkey"
         payload:
-          $ref: "#/components/schemas/EncodedByteArray"
+          $ref: "#/components/schemas/EncodedTx"
```

Comment:
This really breaks it, since this is not what the code does. In this stage, I think we can only accept OAS changes that leave the code as is.

Reply:
Do you mean the payload is not only a transaction?